keyboards/tokyokeyboard/alix40/info.json

```diff
@@ -163,27 +163,27 @@
             },
             {
                 "label": "N",
-                "x": 7.25,
+                "x": 8.25,
                 "y": 2
             },
             {
                 "label": "M",
-                "x": 7.25,
+                "x": 9.25,
                 "y": 2
             },
             {
                 "label": ",",
-                "x": 7.25,
+                "x": 10.25,
                 "y": 2
             },
             {
                 "label": ".",
-                "x": 7.25,
+                "x": 11.25,
                 "y": 2
             },
             {
                 "label": "/",
-                "x": 7.25,
+                "x": 126.25,
```

Comment:
Is `126.25` correct here?  Seems rather large.

Reply:
You're right. That's a typo. It should be 12.25